tests/disaster-recovery/mdr/test_active_hub_down_and_restore.py

```diff
@@ -0,0 +1,282 @@
+
+from ocs_ci.framework.pytest_customization.marks import tier4a, turquoise_squad
+from ocs_ci.framework import config
+from ocs_ci.ocs.acm.acm import AcmAddClusters, validate_cluster_import
+from ocs_ci.ocs.dr.dr_workload import validate_data_integrity
+from ocs_ci.ocs import constants
+from ocs_ci.deployment.deployment import Deployment
+from ocs_ci.ocs.node import get_node_objs
+from ocs_ci.helpers.dr_helpers import (
+    enable_fence,
+    enable_unfence,
+    get_fence_state,
+    failover,
+    relocate,
+    restore_backup,
+    create_backup_schedule,
+    set_current_primary_cluster_context,
+    set_current_secondary_cluster_context,
+    get_current_primary_cluster_name,
+    get_current_secondary_cluster_name,
+    get_passive_acm_index,
+    wait_for_all_resources_creation,
+    wait_for_all_resources_deletion,
+    gracefully_reboot_ocp_nodes,
+    verify_drpolicy_cli,
+)
+from ocs_ci.helpers.dr_helpers_ui import (
+    check_cluster_status_on_acm_console,
+    failover_relocate_ui,
+    verify_failover_relocate_status_ui,
+)
+from ocs_ci.ocs.exceptions import UnexpectedBehaviour
+from ocs_ci.ocs.resources.pod import wait_for_pods_to_be_running
+from ocs_ci.ocs.utils import get_active_acm_index
+from ocs_ci.utility import version
+from ocs_ci.utility.utils import TimeoutSampler
+
+
+logger = logging.getLogger(__name__)
+
+
+@tier4a
+@turquoise_squad
+class TestApplicationFailoverAndRelocateWhenZoneDown:
+    """
+    Test failover and relocate all apps in a single zone after a zone disruption
+    """
+
+    @pytest.fixture(autouse=True)
+    def teardown(self, request, dr_workload):
+        """
+        If fenced, unfence the cluster and reboot nodes
+        """
+
+        def finalizer():
+            if (
+                self.primary_cluster_name
+                and get_fence_state(
+                    drcluster_name=self.primary_cluster_name,
+                    switch_ctx=get_passive_acm_index(),
+                )
+                == "Fenced"
+            ):
+                enable_unfence(
+                    drcluster_name=self.primary_cluster_name,
+                    switch_ctx=get_passive_acm_index(),
+                )
+                gracefully_reboot_ocp_nodes(self.namespace, self.primary_cluster_name)
+
+        request.addfinalizer(finalizer)
+
+    @pytest.mark.polarion_id("OCS-XXXX")
+    def test_application_failover_and_relocate(
+        self,
+        setup_acm_ui,
+        nodes_multicluster,
+        dr_workload,
+    ):
+
+        """
+        Tests to verify failover and relocate all apps in a single zone after a zone disruption
+
+        """
+
+        if config.RUN.get("mdr_failover_via_ui"):
+            ocs_version = version.get_semantic_ocs_version_from_config()
+            if ocs_version <= version.VERSION_4_12:
+                logger.error(
+                    "ODF/ACM version isn't supported for Failover/Relocate operation"
+                )
+                raise NotImplementedError
```

Comment:
We may no longer need this due to release-branches. Kindly check.

Reply:
I'm not sure, it didn't work removing this so kept as it is